code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -603,6 +603,9 @@ Assign your candidates in choose_candidates() instead.
 	var/probability = clamp(threat_level, 30, 100)
 	return !prob(probability)
 
+/datum/dynamic_ruleset/roundstart/extended/choose_candidates()
```

Comment:
Uh ?

Reply:
A one line fix for a runtime. Rulesets that don't actually take any candidates should return TRUE here.